docs/notebooks/introduction.pct.py

```diff
@@ -191,7 +195,7 @@
 # intervals of its predictive distribution.
 
 # %%
-fig = plot_gp_plotly(gpr, mins, maxs, grid_density=30)
+fig = plot_gp_plotly(model['OBJECTIVE'].model, mins, maxs, grid_density=30)
```

Comment:
```suggestion
fig = plot_gp_plotly(model[OBJECTIVE].model, mins, maxs, grid_density=30)
```
someone could rename `OBJECTIVE`, or change the string it points to, and miss this